### PR TITLE
Updated documentation for Altium Live

### PR DESCRIPTION
**Pull request by hannahvsawiuk**

### CODE REVIEW

File: docs/getting-started/cad-software.md [hunk 1]
```diff
@@ -6,34 +6,23 @@ Setup instructions on how install CAD software for schematic capture and PCB lay
...
 ## Altium Setup
+Altium is only compatible with Windows. We currently use **version 18.0**. For more information on the system requirements for this version of Altium, go [here](https://www.altium.com/documentation/altium-designer/altium-designer-system-requirements?version=18.0).
 
-{% hint style="warning" %}
-Altium is only compatible with Windows. We currently use version 16.0. For more information on the system requirements for this version of Altium, go [here](https://www.altium.com/documentation/16.0/display/ADES/Altium+Designer+-+%28%28System+Requirements%29%29).
-{% endhint %}
+### Part 1 - Installation
 
-### ECE Students
+1. Create an [Altium Live account](https://live.altium.com/#Join). Once the request is sent, it may take a few days for approval. Note: If you do not see any entry fields for your information on the linked page, try a different browser.
```
Comment (liambontkes) [inline docs/getting-started/cad-software.md:13]: Is the email address used when creating the account not important? I thought the email address had to granted license permission through ece. This is what I remember doing as an IGEN student.

File: docs/getting-started/cad-software.md [hunk 2]
```diff
@@ -47,26 +36,18 @@ If you do not see fields to enter your information on the linked page, try a dif
 
 ![](../../.gitbook/assets/altium4.PNG)
 
-{% hint style="warning" %}
 If the connection is not successful, try the following:
 
 If you are on campus, ensure that you are connected to **UBCSecure** or **UBCPrivate**. 
 
 If you are using the license remotely, ensure that you are connected to myVPN.
```
Comment (liambontkes) [inline docs/getting-started/cad-software.md:44]: I believe there is a UBC specific vpn service through cisco, can be downloaded through the UBC software downloads website.

File: docs/getting-started/cad-software.md (hunk 1 above)
Comment (hannahvsawiuk) [reply docs/getting-started/cad-software.md:13]: I used my personal email and it was fine. However, I will add that this should be an UBC alum email

File: docs/getting-started/cad-software.md (hunk 2 above)
Comment (hannahvsawiuk) [reply docs/getting-started/cad-software.md:44]: myVPN is the UBC vpn through Cisco :)

**Pull request closed without merging**